publish.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow publishes the scispacy package (not the scispacy models) to pypi.
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Publish Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    
+    if: github.repository == 'allenai/scispacy'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload -u scispacy -p ${{ secrets.PYPI_PASSWORD }} dist/*
```

Comment:
how do secrets work? where are they?

Reply:
If you go to `settings > secrets`, you just specify key value pairs which you can access inside github actions like this. It's quite handy!

Reply:
sweet, that is super nice